modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -103,26 +103,45 @@ def enqueue_email(at, template_id)
 
       # async job and form data includes email
       if email_from_form_data && first_name_from_form_data
+        async_job_with_form_data(email_from_form_data, first_name_from_form_data, at, template_id)
+      # async job and we have a UserAccount
+      elsif user_account
+        async_job_with_user_account(user_account, at, template_id)
+      end
+    end
+
+    def async_job_with_form_data(email, first_name, at, template_id)
+      if Flipper.enabled?(:simple_forms_notification_callbacks)
         VANotify::EmailJob.perform_at(
           at,
-          email_from_form_data,
+          email,
           template_id,
-          get_personalization(first_name_from_form_data)
+          get_personalization(first_name),
+          Settings.vanotify.services.va_gov.api_key,
+          { metadata: { notification_type:, form_number:, statsd_tags: }.to_json }
```

Comment:
@nathanbwright I figured it would be sufficient to just pass in a hash that includes a `metadata` key and nothing else. Does this interface seem reasonable to you?